koku/masu/test/database/test_oci_report_db_accessor.py

```diff
@@ -161,3 +163,19 @@ def test_table_properties(self):
 
     def test_table_map(self):
         self.assertEqual(self.accessor._table_map, OCI_CUR_TABLE_MAP)
+
+    def test_tb_ms_unit(self):
+        """Check that the TB_MS unit is properly converted to GB-Mo."""
+        file = "reports_usage-csv_0001000000829494"
+        file_name = f"{file}.csv"
+        file_path = f"./koku/masu/test/data/oci/{file_name}"
+        with open(file_path) as file:
+            csv = file.readlines()
+            for line in csv:
+                if "TB_MS" in line:
+                    url = reverse("reports-oci-storage")
+                    response = self.client.get(url, **self.headers)
+                    if response.status_code == status.HTTP_200_OK:
+                        unit = response.data["meta"]["total"]["usage"]["units"]
+                        self.assertNotEqual(unit, "TB_MS")
+                        self.assertEqual(unit, "GB-Mo")
```

Comment:
This test file and that API response are completely unrelated. Unfortunately, we don't run the trino-sql anywhere in unit tests, so I would remove this test and coordinate with QE to ensure there is a test case for this.